hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMCertStore.java

```diff
@@ -70,35 +85,69 @@ public void storeValidCertificate(BigInteger serialID,
   }
 
   @Override
-  public void revokeCertificate(BigInteger serialID) throws IOException {
+  public Optional<Long> revokeCertificates(
+      List<BigInteger> serialIDs,
+      X509CertificateHolder caCertificateHolder,
+      CRLReason reason,
```

Comment:
Can we add a revoke time parameter for this interface method so that we don't need to modify the interface later to support scheduled revoke? The caller can pass now() until the scheduled revoke is implemented.